examples/federated-learning/run.py

```diff
@@ -108,14 +109,16 @@ def update_model(self, *grads):
           param.assign(param - grad * self.LEARNING_RATE)
           for param, grad in zip(params, grads)
       ])
-      # return update_op
 
     with tf.name_scope('validate'):
       x, y = self._build_data_pipeline()
       y_hat, loss = self._build_validation_step(x, y)
 
       with tf.control_dependencies([update_op]):
-        return tf.print('expect', loss, y, y_hat, summarize=50)
+        print_loss = tf.print("loss", loss)
+        print_expected = tf.print("expect", y, summarize=50)
+        print_result = tf.print("result", y_hat, summarize=50)
+        return print_loss, print_expected, print_result
```

Comment:
you should probably group these guys (the internal grouping is more a security check that we should not rely on)